### ISSUE TITLE
[sync-branches] generate-changelog is not working correctly

### ISSUE THREAD

**Issue by kenji-miyake** [autowarefoundation/autoware-github-actions#57]
The changelog is empty.
https://github.com/tier4/autoware.universe/runs/5085507144?check_suite_focus=true

**Comment by kenji-miyake** [comment]
Seeing https://github.com/kenji-miyake/autoware-github-actions/pull/16 is working, it's because the repository is different? :thinking: 
https://github.com/kenji-miyake/autoware-github-actions/runs/5085573215?check_suite_focus=true

**Comment by kenji-miyake** [comment]
If I run `git-cliff` manually, it works as expected.

```sh-session
❯ git cliff -vv origin/tier4/main..HEAD
 WARN  git_cliff > "cliff.toml" is not found, using the default configuration.
 DEBUG git_cliff::changelog > Processing the commits...
 DEBUG git_cliff::changelog > Processing the releases...
 DEBUG git_cliff::changelog > Generating changelog...
# Changelog
All notable changes to this project will be documented in this file.

## [unreleased]

### Bug Fixes

- Point cloud conversion logic (#308)
- Fix detection_area findForwardOffsetSegment (#305)
- Remove rosbridge launch from web_controller launch file (#318)
- Fix virtual_traffic_light singed_arc_length and state (#306)
- Suppress warnings  (#298)
- Suppress warnings  (#296)
- Suppress warnings (#293)
- Suppress warnings  (#295)
- Suppress warnings  (#294)
- Changed pointcloud conversion method (#336)
- Add timestamp (#317)
- Memory leak problem (#333)
- Occlusion spot offset (#290)

### Documentation

- Fix invalid links (#309)
- Add obstacle stop doc (#266)

### Features

- Reset previous acceleration when the car starts moving (#303)
- Add instrument type and id to virtual_traffic_light logger (#328)
- Add parameter for tuning tire radius (#327)
- Add particles param (#330)

### Miscellaneous Tasks

- Lighten detection area debug marker (#314)
- Remove path point with lane id (#316)
- Sync files (#312)
- Sync files (#326)
- Sync automatic-rebase.yaml from autowarefoundation/autoware
- Replace legacy timer (#329)
- Update package.xml (#331)
- Fix sync-files.yaml
- Sync community health files (#334)
- Sync files (#335)
- Sync files (#341)
- Sync docs workflows (#342)
- Sync check-build-depends.yaml with autoware_common

### Styling

- Fix English (#307)

### Ci

- Change build-and-test workflow (#297)
- Add sync-files.yaml (#311)
- Add codecov-action (#292)
- Change option (#323)
- Autoupdate (#124)
- Use containers made in the AWF repository (#337)

<!-- generated by git-cliff -->
```
